OpenTrons/opentrons-api
# feat(shared-data): update module offsets for DVT cradles

Closes RSS-204

# Overview

DVT cradles have slightly different measurements than EVT ones. We'll be replacing all existing ones in the office with DVT cradles so need to update the present offsets.

# Test Plan

Make sure to test on an OT3 with DVT cradles and aligned gantry system. Make sure to also complete pipette calibration before starting this test. 
- Upload a protocol with labware loaded on temp module, heater-shaker and thermocycler. Include pipetting steps to/from these modules in the protocol. 
- Run LPC without applying any existing offsets, the pipette should go to the right location on module labware. 
- Ideally there should be no additional LPC offsets required but practically there _might_ be some. Note down these LPC offsets

# Changelog

- updated affine transforms in shared data

# Review requests

- Ideally a code review should be sufficient but from past testing we have found important distinctions between theoretical offset vs actual observed offset so we should probably test this (w/ testing plan above) and note down those differences, if any.
- Make sure I've used the correct offsets mentioned in the excel sheet for DVT cradles. There's tests that verify the resulting offsets, so as long as the final offsets are correct, we should be good.

# Risk assessment

Low.
